src/main/java/build/buildfarm/worker/cgroup/Controller.java

```diff
@@ -22,10 +22,21 @@
 import java.io.Writer;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 abstract class Controller implements IOResource {
   protected final Group group;
 
+  private static final Logger logger = Logger.getLogger(Controller.class.getName());
```

Comment:
We seem to be using a lombok annotator for logging classes these days that might fit better here